packages/embed-examples/src/reactComponent.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { createRoot } from "react-dom/client";
+import GenomeSpy from "@genome-spy/react-component";
+
+const App = () => {
+    const spec = {
+        "data": {
+            "name": "dynamicData"
+        },
+        hconcat: [
+            {
+                height: 250,
+                width: 500,
+                mark: "rect",
+                encoding: {
+                    x: {
+                        field: "id",
+                        type: "nominal",
+                        scale: {
+                            padding: 0.1,
+                        },
+                    },
+                    y: {
+                        field: "y",
+                        type: "quantitative",
+                        scale: { domain: [-1, 1] },
+                    },
+                },
+            },
+        ],
+    }
+
+    const generateData = () => {
```

Comment:
This is overly complex for a simple example with static data. I would provide a flat, static array of objects.

Reply:
Should be fixed in 0ba4f7a